CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+## Contributing to this project
+
+Everybody is encouraged to contribute to this project. If you're part of the [alphagov GitHub organisation](https://www.github.com/alphagov) you probably have write access to this repo. If you don't, ask a GitHub admin or #tech-docs-format on Slack.
```

Comment:
Who is "everybody" here? anyone with a Github account or anyone part of Alphagov?

Reply:
Updated in https://github.com/alphagov/tech-docs-gem/pull/21/commits/3937eb1a08e7a11c863a9b388ec9a720db6a4a0a